cardano-cli/src/Cardano/CLI/Legacy/Run/Genesis.hs

```diff
@@ -1,6 +1,6 @@
 {-# LANGUAGE BangPatterns #-}
-{-# LANGUAGE ConstraintKinds #-}
 {-# LANGUAGE CPP #-}
```

Comment:
I think CPP extension is redundant after that change.

Reply:
Done!